zeppelin-web/pom.xml

```diff
@@ -33,9 +33,9 @@
   <name>Zeppelin: web Application</name>
 
   <!-- See https://github.com/eirslett/frontend-maven-plugin/issues/229 -->
-  <prerequisites>
-    <maven>3.1.0</maven>
-  </prerequisites>
+  <!--<prerequisites>-->
+    <!--<maven>3.1.0</maven>-->
```

Comment:
no need use low maven version for avoid Ubuntu 15.04's problem itself'. https://github.com/eirslett/frontend-maven-plugin/issues/229  , upgrade the frontend-maven-plugin to 1.0